ssl/ssl_transfer_asn1.cc

```diff
@@ -35,8 +35,7 @@ bool ssl_transfer_supported(const SSL *in) {
       !in->s3->established_session ||
       SSL_in_init(in) ||                                        // (4)
       in->version != TLS1_2_VERSION ||                          // (5)
-      in->s3->wnum != 0 ||                                      // (6)
-      in->s3->wpend_tot > 0 ||
+      in->s3->unreported_bytes_written > 0 ||                   // (6)
       in->s3->read_shutdown != ssl_shutdown_none ||             // (7)
```

Comment:
Should this check `in->s3->pending_write.size() > 0` as well?

Reply:
right, added it.